index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,500;1,400&display=swap" rel="stylesheet">
+
+  <title>My Portfolio</title>
+</head>
+<body>
+<header>
+  <nav class="d-flex header">
+    <a href="#" class="logo  nav-link"><span>My Logo</span></a>
+    <span class="material-icons nav-link" >menu</span>
+  </nav>
```

Comment:
- [ ]  Please ensure you have matched the design for the header section. For more details check the screenshot image attached by the previous reviewer.